README.md

```diff
@@ -1,806 +1,10 @@
-# App Functions SDK (Golang) - Beta
 
-Welcome the App Functions SDK for EdgeX. This sdk is meant to provide all the plumbing necessary for developers to get started in processing/transforming/exporting data out of EdgeX. 
+# Application Functions SDK
 
-Table of contents
-=================
+For documentation please visit https://docs.edgexfoundry.org/microservices/application/ApplServices/.
```

Comment:
Change link to `https://docs.edgexfoundry.org/1.2/microservices/application/ApplServices`